content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+---
+title: SPIRE OIDC Federation with AWS
+short: Federation
+description: Setting up SPIRE to federate with AWS
+weight: 50
+toc: true
+menu:
+  spire:
+    weight: 20
+    parent: 'spire-try'
+---
+Since version 0.8.1, SPIRE contains experimental support for [Open ID Connect](https://openid.net/connect/) (OIDC) federation and SPIRE Server-to-SPIRE Server federation. This document describes how to authenticate SPIRE workloads to Amazon AWS S3 buckets using OIDC so the workloads can use data on S3. Using this document as a guide, you should be able to configure access to AWS services other than S3.
```

Comment:
I think this section deserves to be a bit more focused to introduce the problem. For example:

```
This tutorial builds on the Kubernetes Quickstart guide to describe how a SPIRE identified workload can, using a JWT-SVID, authenticate to Amazon AWS APIs, assume an AWS IAM role, and retrieve date from an AWS S3 bucket. This avoids the need to create and deploy AWS IAM credentials with the workload itself.
```